avm/ptn/azd/container-app/main.bicep

```
@@ -233,3 +233,17 @@ output serviceBind object = !empty(serviceType)
 
 @description('The uri of the Container App.')
 output uri string = ingressEnabled ? 'https://${containerApp.outputs.fqdn}' : ''
+
+@description('The resource ID of the Container App.')
+output resourceId string = containerApp.outputs.resourceId
+
+type environmentVar = {
```

```suggestion
type environmentType = {
```

Small suggestion - but would be great as it aligns with the other modules